config/app.php

```diff
@@ -22,11 +22,11 @@
     |
     | This value determines the "environment" your application is currently
     | running in. This may determine how you prefer to configure various
-    | services the application utilizes. Set this in your ".env" file.
+    | services the application utilizes. Set this in your "..env" file.
     |
     */
 
-    'env' => env('APP_ENV', 'production'),
+    '.env' => env('APP_ENV', 'production'),
```

Comment:
ini kok ada penambahan . (dot) sog

Reply:
oiya.. kena refactor kyknya
sep. will fix it